### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 es 4.24 (30 ottobre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#307] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/e1b4fd76-a460-4a47-babc-e9fc2e8b641a)

**Comment by marcoseverini** [comment]
prima parte 
![immagine](https://github.com/user-attachments/assets/e48cfd84-5de9-4246-b34b-895015787abc)
